src/AVFoundation/Enums.cs

```diff
@@ -761,4 +760,18 @@ public enum AVContentKeyRequestStatus : nint
 		Failed
 	}
 
+	public enum AVContentKeyRequestRetryReason {
+		[Field ("AVContentKeyRequestRetryReasonTimedOut")]
+		TimedOut,
+		[Field ("AVContentKeyRequestRetryReasonReceivedResponseWithExpiredLease")]
+		ReceivedResponseWithExpiredLease,
+		[Field ("AVContentKeyRequestRetryReasonReceivedObsoleteContentKey")]
+		ReceivedObsoleteContentKey,
+	}
+
+	public enum AVKeySystem {
+		[TV (10, 2), Mac (10, 12, 4), iOS (10, 3)]
```

Comment:
Since it's a new enum, the availability attributes can be moved to the type itself (as the minimum).
It also seems there should be a `[NoWatch]` there (no attribute will assume `[Watch(2,0)]`)